tests/StaticCaching/FileCacherTest.php

```diff
@@ -74,8 +74,8 @@ public function gets_file_path_from_url()
         ]);
 
         $this->assertEquals(
-            'test/path/foo/bar/baz/qux_a=b&c=d.html',
-            $cacher->getFilePath('http://domain.com/foo/bar/baz/qux?a=b&c=d')
```

Comment:
What was the reason for this change? The test passes even if you leave it how it was.

Is it just to show the query string is not normalized here?

Reply:
Yes 👌🏻